app/models/occupation_standard.rb

```diff
@@ -32,9 +32,18 @@ class OccupationStandard < ApplicationRecord
     mappings dynamic: false do
       indexes :title, type: :text, analyzer: "snowball"
       indexes :ojt_type, type: :text
+      indexes :work_process_titles, type: :text
     end
   end
 
+  def as_indexed_json(_ = {})
+    as_json(
+      include: {work_processes: {only: [:title]}}
+    ).merge(
+      work_process_titles: work_processes.pluck(:title)
+    )
+  end
```

Comment:
This is used by elasticsearch. We're overriding the default method so that we can index extra fields outside the model fields, so we can index associations